spi/src/main/java/org/jboss/pnc/spi/environment/RunningEnvironment.java

```diff
@@ -51,4 +52,9 @@
      */
     RepositorySession getRepositorySession();
 
+    /**
+     * @return Returns a build directory.
+     */
+    Path getWorkingDirectory();
```

Comment:
I think we should. Some build script may require absolute paths. Additionally, unit tests are invoked in temp directory.